bitnami/mongodb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mongodb
-version: 9.0.1
+version: 9.0.2
```

Comment:
Could you bump the minor version? It is adding a feature

Reply:
Of course :+1: